doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -197,6 +210,34 @@ with a ``.ipynb`` file.
 
 .. _MyST-NB: https://myst-nb.readthedocs.io/
 
+.. _interactive-docs:
+
+Interactive examples in docstrings
+----------------------------------
+
+Examples within docstrings can be made interactive using the [``jupyterlite-sphinx`` extension](https://jupyterlite-sphinx.readthedocs.io/en/latest/).
+The buttons for converting examples sections into embedded interactive
+notebooks are hidden by default on clean docs builds. To enable interactive
```

Comment:
This does not seem to be true. I built locally and I do see the buttons, they are just not functional. Should we remove this, or is it an error on jupyterlite-sphinx?

Note that the try_examples.json file is as described below, so theoretically the buttons _should not_ be enabled.

Reply:
If the `try_examples.json` config file is present and is set up  like you've described here through the regex pattern, all the examples should get disabled. If the config file is absent, all examples are enabled everywhere. I checked and the config file is present in the `main` branch, and the interactive examples are disabled on both the stable version https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.linalg.expm.html#scipy.sparse.linalg.expm and on the devdocs https://scipy.github.io/devdocs/reference/generated/scipy.sparse.linalg.expm.html, so this is working, isn't it?

Reply:
@melissawm. Did you mean you see the buttons for the interactive examples in the API reference, or for the notebooks. The config file only impacts the `try_examples` directive and the tutorial markdown notebooks don't use that directive. That could explain things.

Reply:
Yeah - I mean in a clean local docs build, with try_examples.json as 
```
{
    "global_min_height": "400px",
    "ignore_patterns": [".*"]
}
```

I see the buttons. I usually watch the static html files, so my understanding is that the buttons are hidden by JS that doesn't get loaded in the static pages, maybe?

Reply:
I think the JS should get loaded – as the HTML pages generated by Sphinx contain `<script src="_static/jupyterlite_sphinx.js?v=<some kind of ID here>"></script>`, which will attempt to load the `try_examples.json` config file. @steppi is right, the notebook buttons can't be hidden at the moment, this only impacts the API reference examples.

I don't know if it makes sense to add a global configuration for notebook buttons, as they are added in each notebook and are currently not global.